jcl/src/java.base/share/classes/java/lang/ClassLoader.java

```diff
@@ -352,44 +360,35 @@ protected ClassLoader(ClassLoader parentLoader) {
  *@since 9
  */
 protected ClassLoader(String classLoaderName, ClassLoader parentLoader) {
+	this(checkSecurityPermission(), classLoaderName, parentLoader);
+}
+
+private ClassLoader(Void staticMethodHolder, ClassLoader parentLoader) {
```

Comment:
Is this constructor necessary?  Won't this cause the security check to be run multiple times?

ie: 
```
protected ClassLoader(ClassLoader parentLoader) {
   this(checkSecurityPermission(), parentLoader);
}
private ClassLoader(Void staticMethodHolder, ClassLoader parentLoader) {
	this(checkSecurityPermission(), null, parentLoader);
}
private ClassLoader(Void staticMethodHolder,
 /*[IF Sidecar19-SE]*/
 	String classLoaderName,
 /*[ENDIF]*/
 	ClassLoader parentLoader) {
```

If it's only used by this class, can we pass `null` for the classloader name and bypass it?

Reply:
Good catch. Initial thought was to avoid introducing `Java 9` decoration into constructors for `Java 8` as well.
Fixed to remove the constructor in question, and decorate `null` for the classloader name with flag `Sidecar19-SE`. Verified that `pConfig` still compiles.
Please review again.